docs/_includes/projects/index-typescript/custom-greeting.ts

```diff
@@ -0,0 +1,10 @@
+import { LitElement, html, property } from 'lit-element';
+
+export class CustomGreeting extends LitElement {
+  @property({ type: String}) name = 'World';
+  render() {
+    return html`<p>Hello, ${this.name}!</p>`;
+  }
+}
+
+customElements.define('custom-greeting', CustomGreeting);
```

Comment:
let's use the `customElement` decorator too:

```ts
@customElement('custom-greeting')
export class CustomGreeting extends LitElement { 
```

Reply:
Do you also then have to add the `interface HTMLElementTagThingStuff` ?

https://github.com/Polymer/lit-element/blob/0093b439fa53e26707a84690d17bf15f54baae3e/src/lib/decorators.ts#L78

 Seems noisier

Reply:
@katejeffreys No, as in #291 this was changed to only be a string. Before it would actually check if the string was part of `HTMLElementTagNameMap`. Probably the comment in the source code should also be removed.

Reply:
can we then remove `customElements.define('custom-greeting', CustomGreeting);`?

Reply:
@katejeffreys Yes, `@customElement('custom-greeting')` and `customElements.define('custom-greeting', CustomGreeting);` do the same thing. Only one is needed.